cypress/integration/home.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { constants } from '../support';
+
+describe('Home page', () => {
+  beforeEach(() => {
+    cy.visit('/');
+    cy.get('[data-cy="hydrated"]');
```

Comment:
Are you gonna replace this with the solution suggested by Lev? The one at
https://gist.github.com/stevekinney/9fc5c5f69006722739546751df026ceb
I'd prefer it if you do